### ISSUE TITLE
Please add an option to choose whether to display hydrogen used for chirality

### ISSUE THREAD

**Issue by Lo-Dyih-Yuaan** [Acylation/obsidian-chem#56]
### What would you like to be added?

I want the plugin can add an option to choose whether to display hydrogen used for chirality. I think this can be achieved by controlling the `explicitHydrogens` option in JS.

### Why is this needed?

In most cases, the hydrogen in SMILES does not need to be written and displayed, except for those used to indicate chirality. However, displaying these hydrogen separately is not necessary in most cases, and is inconsistent with the treatment of other hydrogen, and making structures appear cluttered.

**Comment by Acylation** [comment]
Yeah will add this in the next update. Thanks for proposing this!

Testcase here:
`N[C@@H](C)C(=O)O`

**Comment by Acylation** [comment]
For the incoming rdkit core, I need find new ways to support this. The imlementations between theses cores are somewhat diffrent.

SmilesDrawer can process `N[C@@H](C)C(=O)O`
RDKit-js will process `C1C[C@@H]2C(C)CCCC2CC1`
